.travis.yml

```diff
@@ -6,7 +6,7 @@ python:
   - 2.7
 
 env:
-  - DEPS="nose numpy scipy matplotlib traits traitsui ipython h5py sympy scikit-learn"
+  - DEPS="nose numpy scipy matplotlib traits traitsui ipython h5py sympy scikit-learn netcdf4"
```

Comment:
Just trying to fix travis errors... For some reason all hdf5 tests on travis fail, while I cannot reproduce it myself. Hence "test" in the commit message